dotCMS/src/integration-test/java/com/dotcms/graphql/business/GraphqlAPITest.java

```diff
@@ -626,6 +631,77 @@ public void testGetSchema_BothSidedRelationshipFields(final RELATIONSHIP_CARDINA
 
     }
 
+    @DataProvider
+    public static List<Object> dataProviderEEContentTypes() throws Exception {
+        // data provider needs stuff to get initialized because of API access
+        IntegrationTestInitService.getInstance().init();
+
+        // filter only Enterprise content types
+        List<ContentType> eeTypes = APILocator
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'eeTypes' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4740432)